less/components/btn.less

```diff
@@ -1,4 +1,8 @@
 .btn {
+  // button text should be wrapped.
+  // we shouldn't assume button text can always fit into one line,
+  // especially knowing that the site will be localized in the future
```

Comment:
I think a more future-proof way to write this would be "especially knowing that the site needs to be localizable". One of the downsides of comments is that it's very easy for them to become out-of-date (no unit tests for comments lol), so odds are this comment will still exist far into the future when our site has been localized :)

Not a big deal though, I doubt it'd cause much confusion to folks reading it in the future... Just generally good practice (IMO) to think about what might happen in the future if the comment is left unchanged, and therefore to embed as few assumptions in the comment as possible, you know?